OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
+- cpmeadors
+- asood-rh
+- paigerube14
+reviewers:
+- cpmeadors
+- asood-rh
+- paigerube14
+- arnaik-rh
```

Comment:
@paigerube14 Could get rid of the extra line/character?

Reply:
there isn't a new line at the bottom there. I think its  saying there's no new line at the end

Reply:
![owners](https://github.com/openshift-qe/ocpqe-security-tools/assets/64206430/0608be74-0590-4e83-9b91-5efbfd6d45eb)

Reply:
Just a nit, Add a new line then. Rest looks good.

/lgtm